Sources/Nuke/Internal/Extensions.swift

```diff
@@ -6,17 +6,17 @@ import Foundation
 import CryptoKit
 
 extension String {
-    /// Calculates SHA1 from the given string and returns its hex representation.
+    /// Calculates SHA256 from the given string and returns its hex representation.
     ///
     /// ```swift
-    /// print("http://test.com".sha1)
-    /// // prints "50334ee0b51600df6397ce93ceed4728c37fee4e"
+    /// print("http://test.com".256)
```

Comment:
should it be `/// print("http://test.com".sha256)`?

Reply:
oof good catch. let me revise